app/components/subcases/subcase-item.js

```diff
@@ -42,6 +65,11 @@ export default class SubcaseItemSubcasesComponent extends Component {
     this.hasDocumentsToShow = doc !== null;
   }
 
+  @task
+  *loadRelatedMeeting() {
+    this.requestedForMeeting = yield this.args.subcase.requestedForMeeting;
```

Comment:
Waarom deze aparte load-task/tracked property en niet gewoon in de template laten resolven met `@subcase.requestedForMeeting`? Dit wordt toch maar op 1 plaats gebruikt?

Reply:
hmmm, true.
Ik heb daar een awaited computed vervangen en in die mindset heb ik een tracked gebruikt. Die tracked is op zich niet nodig, gewoon data loading doen bij constructie en dan in de template aanspreken.

Reply:
fixed it